overlord/configstate/configcore/journal.go

```diff
@@ -45,33 +49,36 @@ func handleJournalConfiguration(tr config.ConfGetter, opts *fsOnlyContext) error
 		return nil
 	}
 
-	var storage string
 	if output == "" {
 		return nil
 	}
-	switch output {
-	case "true":
-		storage = "persistent"
-	case "false":
-		storage = "auto"
-	default:
-		return fmt.Errorf("unsupported journal.persistent option: %q", output)
-	}
 
 	var sysd systemd.Systemd
 	rootDir := dirs.GlobalRootDir
 	if opts != nil {
 		rootDir = opts.RootDir
 	}
 
-	confDir := filepath.Join(rootDir, "/etc/systemd/journald.conf.d/")
-	if err := os.MkdirAll(confDir, 0775); err != nil {
-		return err
-	}
-	confFile := filepath.Join(confDir, "00-snap-core.conf")
-	content := fmt.Sprintf("[Journal]\nStorage=%s\n", storage)
-	if err := osutil.AtomicWriteFile(confFile, []byte(content), 0644, 0); err != nil {
-		return err
+	logPath := filepath.Join(rootDir, "/var/log/journal")
+
+	switch output {
+	case "true":
+		if err := os.MkdirAll(logPath, 0755); err != nil {
```

Comment:
On my system /var/log is 0755, but systemd-journal is actually 02755 (S_ISGID is set). 
There's also some ACLs set for the `adm` group:
```
$ guest@ubuntu:~$ getfacl /var/log/journal
getfacl: Removing leading '/' from absolute path names
# file: var/log/journal
# owner: root
# group: systemd-journal
# flags: -s-
user::rwx
group::r-x
group:adm:r-x
mask::r-x
other::r-x
default:user::rwx
default:group::r-x
default:group:adm:r-x
default:mask::r-x
default:other::r-x
```